build/linux_package_cpanel.gypi

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2010 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+{
+  'type': 'none',
+  'dependencies': [
+    'all.gyp:All',
+    'linux_installer_configs',
+  ],
+  'actions': [
+    {
+      'action_name': 'linux_package_deb_<(channel)_action',
+      'process_outputs_as_sources': 1,
+      'inputs': [
+        '<(cpanel_build)',
+        '<(PRODUCT_DIR)/install/rpm/mod-pagespeed.spec.template',
+        '<@(packaging_files_binaries)',
+        '<@(packaging_files_common)',
+        '<@(packaging_files_cpanel)',
+      ],
+      'outputs': [
+        '<(PRODUCT_DIR)/ea-apache24-mod_pagespeed-<(channel)-<(version)-r<(revision).<(rpm_arch).rpm',
+      ],
+      'action': [ '<@(rpm_cmd)', '-c', '<(channel)', '-p' ],
+    },
+  ],
+}
+
+# Local Variables:
```

Comment:
There's no formatting hints included in any sourcecode of this repo, so this would be an exception.
Could you remove this? (If we'd include this, I'd prefer it as a project-wide change / guideline. But then it would make sense to first discuss this in a public thread so everyone has a chance to share their opinion/thoughts. So for now, leaving it out would probably  be easiest).

Reply:
Sure. It's worth noting that this file was based off [build/linux_package_rpm.gypi](https://github.com/pagespeed/mod_pagespeed/blob/93541308744ebc586578e531cb54072b0ddb1eda/build/linux_package_rpm.gypi#L30) which includes the formatting hints.

Reply:
I see, looks good to me then. (Perhaps we should clean up the inconsistency at some point in the future, and either include this in all .gypi files, or axe it out everywhere).